runtime/compiler/runtime/J9Runtime.hpp

```diff
@@ -62,29 +62,29 @@ void replaceFirstTwoBytesWithData(void *startPC, int32_t startPCToData);
 
 #if defined(TR_HOST_POWER)
 #define  OFFSET_REVERT_INTP_PRESERVED_FSD                (-4)
-#define  OFFSET_REVERT_INTP_FIXED_PORTION                (-12-2*sizeof(intptr_t))
+#define  OFFSET_REVERT_INTP_FIXED_PORTION                (-12 - 2 * (int32_t)sizeof(intptr_t))
```

Comment:
to be more consistent, i would suggest (-(12 + 2 * sizeof(intptr_t)))

Reply:
I had an offline conversation motivating this change which I should have explained, my apologies.

The problem here was that `OFFSET_REVERT_INTP_FIXED_PORTION` was being passed into a function expecting a `int32_t`, but `sizeof` returns an unsigned number of "at least 16 bits". Because of that, depending on the compiler, `OFFSET_REVERT_INTP_FIXED_PORTION` may have type `uint16_t` which could cause truncation problems. Casting the result of `sizeof` to `int32_t` will force the type of `OFFSET_REVERT_INTP_FIXED_PORTION` to be `int32_t` as well.

Reply:
I was clearly out of it yesterday and was very confused about what you meant, but I made that change, thank you!